### ISSUE TITLE
Support increasing/decreasing font sizes via browser settings

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#3522]
If a user sets bigger or smaller font sizes in the settings of his browser, the changes should be also reflected in the web UI.

**Comment by LukasHirt** [comment]
Needs to be taken care of that the layout is also not completely broken when that happens.

**Comment by kulmann** [comment]
Respecting the font size set in the browser has been implemented in https://github.com/owncloud/web/pull/7363 - we scale the browser font size down to 88% and use that as base font size for all font size related css classes / stylings. Works pretty well for me. Happy to get feedback to improve this further - please reopen this ticket as needed.

Also, @Deaddy the linked PR reduced the UI sizings quite a bit, so that the UI doesn't appear so clunky anymore.
